### DISCUSSION TITLE
Send a message to multiple groups at the same time

### DISCUSSION THREAD

**Discussion by YosefAdPro** [devlikeapro/waha#697] [Q&A] [answered]
Hi!
Is there a way to send a message to multiple groups at the same time by separating them with a **;** or something like that?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment, marked as answer]
nope, just use for loop and send multiple request. 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
